torch_xla/experimental/pjrt.py

```diff
@@ -214,3 +215,20 @@ def run_multiprocess(fn: Callable[..., R], *args,
     }
 
   return results
+
+@requires_pjrt
+def broadcast_xla_master_model_param(model: xm) -> None:
```

Comment:
Nit: I think this name should be more terse. I think `broadcast_master_parameters`. `xla` and `model` are already apparent from the signature and context.

Also, the argument should be type `nn.Module`. `xm` is a module in this context